src/main/resources/db.properties

```diff
@@ -0,0 +1,11 @@
+# PostgreSQL properties
+db.driver=org.postgresql.Driver
+db.url=jdbc:postgresql://localhost:5432/cinema
+db.username=root
+db.password=1111
+# Hibernate properties
+hibernate.dialect=org.hibernate.dialect.PostgreSQL10Dialect
+hibernate.show_sql=false
```

Comment:
isn't it false by default?

Reply:
Yes. The reason - it will be easy to tune if you want to switch value.